frontend/store/auctions.ts

```diff
@@ -235,7 +235,10 @@ export const actions = {
             clearInterval(updateAuctionsPricesIntervalId);
         }
         refetchIntervalId = setInterval(() => dispatch('update'), REFETCH_INTERVAL);
-        updateAuctionsPricesIntervalId = setInterval(() => dispatch('updateAuctionsPrices'), TIMER_INTERVAL);
+        updateAuctionsPricesIntervalId = setInterval(
+            async () => await dispatch('updateAuctionsPrices'),
+            TIMER_INTERVAL
+        );
```

Comment:
yes, buuuuut i have an everlasting problem with forgetting to await sometimes. Aaand making the async syntax propagated to the top helps a bit because it explicitly screams of promises.